packages/progress-ring/package.json

```diff
@@ -13,11 +13,12 @@
   },
   "main": "build/index.js",
   "module": "build/index.es.js",
-  "style": "build/index.css",
   "files": [
     "build/*"
   ],
   "dependencies": {
+    "@hig/theme-context": "^2.0.0",
+    "@hig/theme-data": "^1.5.0",
     "classnames": "^2.2.5",
```

Comment:
I don't think this is necessary anymore, same w/ @hig/styles since.

Also wondering if this should get pre-bumped to `1.0.0` as `notifications-flyout` has this as a dependency.